src/API/FileController.ts

```diff
@@ -0,0 +1,91 @@
+import * as FormData from 'form-data'
+import * as fs from 'fs'
+import { inject, injectable } from 'inversify'
+import { Collection, Db } from 'mongodb'
+import fetch, {Response} from 'node-fetch'
+import * as Pino from 'pino'
+import { map } from 'ramda'
+
+import { childWithFileName } from 'Helpers/Logging'
+import { minutesToMiliseconds  } from 'Helpers/Time'
+import { asyncPipe } from 'Helpers/asyncPipe'
+
+interface IPFSFileResponseJson {
+  readonly Hash: string
+}
+
+interface FileResponseJson {
+  readonly hash: string
+}
+
+enum LogTypes {
+  info = 'info',
+  trace = 'trace',
+  error = 'error',
+}
+
+export interface FileControllerConfiguration {
+  readonly ipfsUrl: string
+}
+
+export const convertJson = (x: IPFSFileResponseJson) => ({
```

Comment:
Does this return a FileResponseJson?

Reply:
Yup, thx, looks like I forgot to add the return type